apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1785,6 +1850,113 @@ static void libc_stdio_fileno_tc(void)
 }
 
 /**
+* @testcase         libc_stdio_remove_tc
+* @brief            Deletes the file whose name is specified in filename.
+* @scenario         Open file
+* @apicovered       remove
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_remove_tc(void)
+{
+	char *filename = VFS_FILE_PATH;
+	int ret = NULL;
+
+	ret = remove(filename);
+	TC_ASSERT_EQ("remove", ret, 0);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_setbuf_tc
+* @brief            Set stream buffer
+* @scenario         Open file
+* @apicovered       setbuf, setvbuf
+* @precondition     NA
+* @postcondition    NA
+*/
+#if CONFIG_STDIO_BUFFER_SIZE > 0
+static void libc_stdio_setbuf_tc(void)
```

Comment:
Some logics for checking whether buffer is set to fp is needed.
For example, check 'whether fp->fs_bufstart is NULL or not' for setbuf(fp, NULL);

Reply:
done